### PR TITLE
Remove unnecessary code

### PR DESCRIPTION
**Pull request by prodigion**
Remove unnecessary code:
* IE specific error checks
* Whitespace
* Relative links and including hostname
* Script type
* topbar

### CODE REVIEW

**Review by dkpat12 — APPROVED** [review]

File: _footer.php [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-<div id="footer"><div style="float: left;"><a href="../legal.php">Legal</a></div>
-<a href="http://niwanetwork.org">Nintendo Independent Wiki Alliance</a> &copy; 2010-<?php echo(date('Y')); ?>
+<div id="footer"><div style="float: left;"><a href="/legal.php">Legal</a></div>
+<a href="/">Nintendo Independent Wiki Alliance</a> &copy; 2010-<?php echo(date('Y')); ?>
 </div>
 
 </div>
 
-<script type="text/javascript" src="https://apis.google.com/js/plusone.js"></script>
+<script src="https://apis.google.com/js/plusone.js"></script>
```
Comment (dkpat12) [inline _footer.php:9]: Was this from Google+ ? If so it can definitely be killed

**Review by dkpat12 — APPROVED** [review]

**Review by KevinPayravi — APPROVED** [review]
Thank you!

File: _footer.php (hunk 1 above)
Comment (KevinPayravi) [reply _footer.php:9]: Sounds like it. @prodigion can you remove this?

Comment (KevinPayravi) [reply _footer.php:9]: Off-site discussion, will be done in different PR.

**Pull request merged**